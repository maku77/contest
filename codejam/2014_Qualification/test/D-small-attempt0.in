50
5
0.73 0.55 0.82 0.91 0.64
0.27 0.36 0.09 0.45 0.18
10
0.95 0.25 0.14 0.82 0.32 0.43 0.59 0.80 0.34 0.64
0.50 0.66 0.11 0.55 0.75 0.16 0.52 0.05 0.09 0.77
10
0.13 0.42 0.74 0.61 0.48 0.94 0.29 0.23 0.32 0.77
0.19 0.71 0.06 0.87 0.81 0.97 0.84 0.58 0.03 0.52
10
0.807 0.597 0.681 0.874 0.824 0.958 0.891 0.756 0.647 0.908
0.353 0.151 0.496 0.521 0.538 0.076 0.513 0.580 0.034 0.092
7
0.83 0.06 0.29 0.63 0.74 0.46 0.03
0.80 0.86 0.69 0.71 0.31 0.91 0.94
8
0.24 0.45 0.07 0.86 0.52 0.55 0.76 0.34
0.31 0.41 0.72 0.14 0.62 0.59 0.48 0.38
6
0.23 0.27 0.06 0.02 0.08 0.21
0.58 0.85 0.42 0.79 0.96 0.29
6
0.12 0.24 0.03 0.47 0.59 0.41
0.65 0.79 0.88 0.74 0.94 0.82
10
0.750 0.296 0.593 0.120 0.694 0.176 0.009 0.648 0.731 0.102
0.278 0.213 0.870 0.259 0.778 0.333 0.352 0.843 0.528 0.157
5
0.76 0.67 0.24 0.71 0.43
0.81 0.05 0.86 0.14 0.33
10
0.59 0.02 0.80 0.73 0.52 0.68 0.32 0.36 0.77 0.45
0.61 0.89 0.25 0.64 0.95 0.05 0.57 0.09 0.82 0.50
10
0.69 0.62 0.04 0.64 0.05 0.07 0.29 0.56 0.45 0.40
0.55 0.85 0.09 0.67 0.11 0.84 0.38 0.71 0.15 0.98
10
0.83 0.55 0.79 0.17 0.04 0.23 0.09 0.72 0.68 0.32
0.91 0.87 0.45 0.26 0.36 0.02 0.64 0.13 0.70 0.30
1
0.2
0.8
10
0.19 0.92 0.48 0.79 0.42 0.46 0.94 0.62 0.37 0.25
0.83 0.10 0.50 0.35 0.38 0.85 0.31 0.12 0.54 0.33
10
0.176 0.029 0.500 0.235 0.706 0.980 0.833 0.765 0.392 0.333
0.186 0.814 0.824 0.843 0.461 0.314 0.196 0.088 0.912 0.225
10
0.614 0.974 0.965 0.658 0.798 0.675 0.737 0.684 0.991 0.807
0.579 0.395 0.281 0.298 0.474 0.500 0.254 0.307 0.272 0.491
1
0.33
0.75
8
0.72 0.69 0.62 0.52 0.60 0.78 0.86 0.82
0.42 0.29 0.25 0.51 0.22 0.34 0.19 0.05
6
0.95 0.29 0.33 0.90 0.76 0.67
0.89 0.03 0.22 0.65 0.78 0.51
10
0.769 0.404 0.067 0.240 0.260 0.471 0.846 0.875 0.144 0.346
0.423 0.058 0.990 0.577 0.683 0.077 0.808 0.962 0.731 0.952
1
0.67
0.83
10
0.07 0.95 0.57 0.24 0.84 0.02 0.87 0.78 0.42 0.76
0.10 0.97 0.75 0.23 0.96 0.33 0.74 0.88 0.28 0.09
10
0.07 0.02 0.14 0.39 0.77 0.54 0.46 0.33 0.81 0.84
0.05 0.72 0.53 0.44 0.91 0.63 0.18 0.25 0.98 0.56
10
0.72 0.74 0.81 0.15 0.86 0.49 0.58 0.65 0.91 0.40
0.79 0.26 0.68 0.59 0.36 0.42 0.85 0.27 0.33 0.96
10
0.892 0.649 0.703 0.414 0.802 0.775 0.721 0.820 0.243 0.595
0.333 0.658 0.955 0.144 0.982 0.315 0.541 0.270 0.135 0.856
10
0.51 0.62 0.65 0.49 0.40 0.15 0.04 0.25 0.55 0.69
0.75 0.67 0.29 0.96 0.56 0.11 0.53 0.82 0.44 0.13
2
0.28 0.67
0.11 0.39
10
0.52 0.79 0.37 0.87 0.56 0.19 0.24 0.53 0.98 0.32
0.73 0.10 0.44 0.34 0.58 0.48 0.42 0.71 0.03 0.69
3
0.33 0.20 0.27
0.07 0.47 0.67
10
0.504 0.761 0.983 0.829 0.487 0.718 0.949 0.752 0.641 0.427
0.385 0.043 0.222 0.085 0.299 0.128 0.265 0.060 0.068 0.077
10
0.22 0.07 0.56 0.41 0.04 0.37 0.59 0.89 0.19 0.96
0.67 0.93 0.78 0.63 0.74 0.85 0.33 0.81 0.44 0.26
9
0.09 0.34 0.03 0.41 0.22 0.06 0.44 0.25 0.47
0.56 0.91 0.53 0.66 0.59 0.84 0.78 0.94 0.81
10
0.33 0.90 0.38 0.62 0.71 0.19 0.29 0.52 0.05 0.48
0.43 0.14 0.81 0.57 0.76 0.10 0.67 0.24 0.95 0.86
5
0.07 0.93 0.04 0.79 0.57
0.11 0.43 0.71 0.61 0.39
10
0.73 0.12 0.38 0.27 0.79 0.06 0.81 0.54 0.85 0.69
0.92 0.58 0.50 0.13 0.25 0.40 0.96 0.42 0.31 0.94
10
0.407 0.667 0.861 0.194 0.204 0.500 0.259 0.056 0.481 0.370
0.870 0.167 0.491 0.731 0.157 0.796 0.519 0.574 0.981 0.944
9
0.87 0.70 0.57 0.61 0.83 0.91 0.48 0.74 0.96
0.35 0.30 0.13 0.43 0.39 0.09 0.26 0.17 0.04
7
0.41 0.30 0.81 0.22 0.17 0.43 0.52
0.21 0.86 0.92 0.87 0.60 0.37 0.54
9
0.38 0.79 0.85 0.59 0.83 0.58 0.02 0.09 0.56
0.64 0.30 0.45 0.61 0.88 0.23 0.65 0.36 0.39
9
0.91 0.98 0.43 0.35 0.15 0.78 0.07 0.30 0.93
0.39 0.48 0.22 0.80 0.83 0.76 0.24 0.37 0.50
10
0.05 0.19 0.71 0.77 0.53 0.63 0.47 0.22 0.54 0.92
0.42 0.89 0.90 0.37 0.52 0.30 0.62 0.13 0.91 0.85
10
0.16 0.30 0.05 0.38 0.62 0.46 0.41 0.14 0.57 0.27
0.84 0.73 0.97 0.70 0.92 0.95 0.81 0.65 0.76 0.68
1
0.1
0.9
1
0.6
0.4
10
0.13 0.78 0.10 0.21 0.17 0.40 0.58 0.16 0.26 0.67
0.83 0.54 0.20 0.73 0.04 0.80 0.30 0.51 0.97 0.24
10
0.77 0.31 0.38 0.73 0.08 0.58 0.96 0.69 0.35 0.19
0.65 0.92 0.15 0.54 0.27 0.12 0.62 0.85 0.50 0.46
10
0.27 0.73 0.87 0.13 0.54 0.78 0.42 0.46 0.91 0.23
0.34 0.24 0.37 0.32 0.15 0.89 0.65 0.48 0.19 0.59
8
0.27 0.31 0.37 0.46 0.01 0.42 0.45 0.03
0.90 0.75 0.48 0.58 0.69 0.63 0.49 0.73
10
0.12 0.70 0.39 0.37 0.05 0.86 0.14 0.04 0.07 0.28
0.35 0.11 0.77 0.32 0.60 0.74 0.95 0.91 0.67 0.79
