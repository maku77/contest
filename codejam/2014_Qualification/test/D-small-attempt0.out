Case #1: 5 5
Case #2: 9 3
Case #3: 7 2
Case #4: 10 10
Case #5: 3 0
Case #6: 7 2
Case #7: 0 0
Case #8: 0 0
Case #9: 7 2
Case #10: 3 2
Case #11: 7 2
Case #12: 6 0
Case #13: 8 1
Case #14: 0 0
Case #15: 10 3
Case #16: 8 2
Case #17: 10 10
Case #18: 0 0
Case #19: 8 8
Case #20: 6 2
Case #21: 6 1
Case #22: 0 0
Case #23: 8 2
Case #24: 8 1
Case #25: 9 3
Case #26: 8 4
Case #27: 7 1
Case #28: 2 1
Case #29: 9 3
Case #30: 1 1
Case #31: 10 10
Case #32: 5 1
Case #33: 0 0
Case #34: 7 1
Case #35: 3 2
Case #36: 7 2
Case #37: 5 1
Case #38: 9 9
Case #39: 3 0
Case #40: 7 2
Case #41: 7 3
Case #42: 7 1
Case #43: 0 0
Case #44: 0 0
Case #45: 1 1
Case #46: 7 1
Case #47: 8 2
Case #48: 9 3
Case #49: 0 0
Case #50: 5 0
